src/main/java/adris/altoclef/butler/ButlerConfig.java

```diff
@@ -51,4 +51,19 @@ public static ButlerConfig getInstance() {
      */
     public boolean whisperFormatDebug = false;
 
+    /**
+     * If set to true, will reply with "Sorry, you're not authorized!".
+     *
+     * Disable this if you need to stay undercover.
+     */
```

Comment:
As the message is customizable, we should avoid putting it in the javadoc. I understand however that javadox vary from project to project. Please replace this javadoc with the following

/**
\* Determines if failure messages should be sent to a non-authorized entity attempting to use butler
\*
/*